tools/ci/tc/tests/test_valid.py

```diff
@@ -71,3 +73,74 @@ def test_verify_payload():
             except Exception as e:
                 print("Validation failed for task '%s':\n%s" % (name, json.dumps(task_data, indent=2)))
                 raise e
+
+
+@pytest.mark.xfail(sys.version_info.major == 2,
+                   reason="taskcluster library has an encoding bug "
+                   "https://github.com/taskcluster/json-e/issues/338")
```

Comment:
We should expect this to start failing on master at some point, given we don't pin json-e, right? 🙁

Reply:
I agree that we should pin all the requirements in a requirements.txt fle, but I don't think that should block this.